### ISSUE TITLE
incorrect handling of &lt;FOO&gt;

### ISSUE THREAD

**Issue by jmdyck** [bterlson/ecmarkup#38]
Running ecmarkup v2.0.1 on
```
<emu-clause>
<h1>blah</h1>
<p>U+200C (ZERO WIDTH NON-JOINER) is abbreviated "&lt;ZWNJ&gt;"</p>
</emu-clause>
```
yields (in part):
```
<p>U+200C (ZERO WIDTH NON-JOINER) is abbreviated "<zwnj>"</zwnj></p>
```

**Comment by bterlson** [comment]
> Running ecmarkup v2.0.1

Huzzah!!! :fireworks: :party: :balloon:

Will look into this. I hate these kinds of issues :(

**Comment by bterlson** [comment]
Fixed in 2.0.2.

This was another one of those replacing-text-node-with-html escaping bugs. Dealt with it in emd by escaping parseFragment but had to handle it for autolinking as well. Probably EMD should get out of the business of escaping HTML output actually considering it's textContent that breaks it. EMD should just expect HTML input with proper escaping.
